managers/parsers/inTechOpenParser.py

```diff
@@ -23,35 +23,39 @@ def createLinks(self):
 
             manifestJSON = self.generateManifest(self.uri, manifestURI)
 
+            htmlURI = self.createHTMLURI()
+
+            # TODO
+            # 1. Add regex to extract page identifier from PDF (e.g. r'intechopen.com\/books\/([\d]+)')
+            # 2. Construct HTML URI using extracted identifier
+            # 3. Add HTML URI to return array below
+            #
+            # QUESTIONS
+            # 1. Do all Intechopen books have PDF links?
+            # 2. How do we handle potential errors/add a fallback condition
+            # 3. Are PDF links formatted consistently?
+
             return [
                 (manifestURI, {'reader': True}, 'application/webpub+json', (manifestPath, manifestJSON), None),
-                (self.uri, {'reader': False, 'download': True}, self.mediaType, None, None)
+                (self.uri, {'reader': False, 'download': True}, self.mediaType, None, None),
+                (htmlURI, {'reader': False}, 'text/html', self.mediaType, None, None)
             ]
-        elif self.mediaType == 'application/epub+zip':
-            ePubDownloadPath = 'epubs/{}/{}.epub'.format(self.source, self.identifier)
-            ePubDownloadURI = '{}{}'.format(s3Root, ePubDownloadPath)
-
-            ePubReadPath = 'epubs/{}/{}/META-INF/container.xml'.format(self.source, self.identifier)
-            ePubReadURI = '{}{}'.format(s3Root, ePubReadPath)
+        elif self.mediaType == 'text/html':
+            return []
 
-            webpubReadPath = 'epubs/{}/{}/manifest.json'.format(self.source, self.identifier)
-            webpubReadURI = '{}{}'.format(s3Root, webpubReadPath)
+        return []
 
-            return [
-                (webpubReadURI, {'reader': True}, 'application/webpub+json', None, None),
-                (ePubReadURI, {'reader': True}, 'application/epub+xml', None, None),
-                (ePubDownloadURI, {'download': True}, self.mediaType, None, (ePubDownloadPath, self.uri))
-            ]
 
-        elif self.mediaType == 'text/html':
-            htmlPath = 'text/{}/{}.html'.format(self.source, self.identifier)
-            htmlURI = '{}{}'.format(s3Root, htmlPath)
+    def createHTMLURI(self):
+        '''Using regular expressions to search for identifier in PDF and parse it into HTML URI'''
+        
+        identRegex = r'intechopen.com\/storage\/books\/([\d]+)'
+        match = re.search(identRegex, self.uri)
+        identifier = match.group(1) 
```

Comment:
This might raise an error if the regex search fails. We should have a handler for that, and then also in the method above make sure we don't add a link object if  `htmlURI` is `None`